.github/CODEOWNERS

```diff
@@ -806,6 +806,9 @@
 # AzureSdkOwners:                                                  @ShivangiReja
 # ServiceOwners:                                                   @arv100kri @bleroy @tjacobhi
 
+# ServiceLabel: %SelfHelp
```

Comment:
Please move this down to the management section around L957, preserving alphabetical ordering by tags.  You'll need to include both the PR and Service ownership and should follow the existing patterns there.

As there is no existing label, please use "Self Help", which I have created.

Reply:
fixed